_includes/script.html

```diff
@@ -1,8 +1,13 @@
 <script type="text/javascript">
     $('h1, h2, h3, h4, h5, h6').filter('[id]').each(function () {
-        var el = document.createElement('a')
-        el.className = 'anchor'
+        var el = document.createElement('a');
+        el.className = 'anchor';
         el.href = '#' + this.id;
         this.prepend(el)
     });
+    $('pre code').each(function () {
+        this.innerHTML = this.innerHTML
+            .split('').map(c => c === '━' || c === '─' ? `<span>${c}</span>` : c)
```

Comment:
I'm not sure this will work for lines containing lots of ┷/┯, etc.

It seems to me that the right solution would be to use a font where these characters have the correct width.

Reply:
`┯ ┷ ┼` have the same width as `│`, so it still aligns properly. I'd like to keep the number of elements low.

Reply:
> It seems to me that the right solution would be to use a font where these characters have the correct width.

I tried several monospaced fonts, and their box-drawing characters were always rendered too wide. This doesn't seem to be the font's fault. I verified that Fira Code includes box-drawing characters, and that they have the same width as other characters. Yet they are rendered too wide on Android. I didn't try it on Windows.

Reply:
That clearly looks like Android is overriding the font definition and replacing it with a system font. How is the Source Code Pro font being provided? I noticed that the OpenSans definitions in [_sass/fonts.scss](https://github.com/nushell/nushell.github.io/blob/master/_sass/fonts.scss) point to source files in a `fonts/` directory, but that directory doesn't exist. Maybe we need to actually include the font files in the repo so that they are actually used when they're not immediately available in the system?

Reply:
I tried it even with Google fonts, so I'm pretty sure that the font was loaded.

The fonts directory is in `_site/assets`. I don't know how that was generated, since `_site` is in `.gitignore`, but I'm not that familiar with jekyll.